src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/FiltersForm.test.tsx

```diff
@@ -376,6 +446,28 @@ describe('FiltersForm', () => {
     ).not.toHaveAttribute('checked');
   });
 
+  test('automatically selects checkbox when there is only one indicator group with one option', () => {
+    render(
+      <FiltersForm
+        {...testWizardStepProps}
+        subjectMeta={testSubjectMetaOneIndicator}
+        onSubmit={noop}
+      />,
+    );
+
+    expect(
+      within(
+        screen.getByRole('group', {
+          name: 'Indicators - 1 selected',
+        }),
+      ).getAllByRole('checkbox'),
+    ).toHaveLength(1);
+
+    expect(
+      screen.getByLabelText('Number of excluded sessions'),
```

Comment:
Same as above - might want to check this is the same checkbox as the one within the indicator group